tutorials/14-minituring/Project.toml

```diff
@@ -0,0 +1,7 @@
+[deps]
+AbstractMCMC = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
```

Comment:
Can you add a `[compat]` section with compat entries for the different packages as well? That makes updating and keeping track of breaking releases of these dependencies easier.

Reply:
`[compat]` section added.